_test/test_unique_objects_container/thingy.m

```diff
@@ -1,4 +1,4 @@
-classdef thingy
+classdef thingy < serializable
```

Comment:
Do these want to go in the general `common_functions` folder with more general names so we can use them globally in tests and keep the testing folder cleaner?

Reply:
The class looks pretty specific to its location to me. I have added a comment about why it is now subclassed